pkg/appgw/appgw_test.go

```diff
@@ -1,680 +0,0 @@
-package appgw
```

Comment:
I see we have added a lot of unit-tests. But where have moved these tests out?

Reply:
I actually deleted `pkg/appgw/appgw_test.go` and `pkg/k8scontext/k8scontext_test.go`
These were *not* refactored at all. I added them back.
Apologies - my bad and thanks for calling this out!
Good thing the PR is just a demo / WIP !